home.html

```diff
@@ -0,0 +1,139 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="home.css" rel="stylesheet">
+   <title>Home page</title>
+</head>
+<body> 
+    <header>
+      <div class="top">
+        <a class="facebook"><img src="./images/facebook.png"></a>
+        <a><img class  ="twitter" src="./images/twitter.png"></a>
+        <a class="Language"><p>English</p></a>
+        <a class="Page"><p>My Page</p></a>
+        <a class="checkout"><p>Logout</p></a>
+      </div>
+        <h1 class="background-img"></h1>
+        <h1 class="h-background-img"></h1>
+        <div class="top-strip"></div> 
+       
+        <h1 class="logo"><img class="my_logo" src="./images/logo.png"> </h1>
+        <nav class="navbar">  
+            <div class="hamburger">
+                <span class="nav_bar"></span>
+                <span class="nav_bar"></span>
+                <span class="nav_bar"></span>   
+            </div>
+            <div class="top-bar">   
+        <ul class="nav_menu">
+            <li class="nav-link"><a href="about.html" class="nav_link">About</a></li>
+            <li class="nav-link"><a href="./" class="nav_link">Program</a></li>
+            <li class="nav-link"><a href="./" class="nav_link">Join</a></li>
+            <li class="nav-link"><a href="./" class="nav_link">Sponsor</a></li>
+            <li class="nav-link"><a href="./" class="nav_link">News</a></li>
+        </ul>
+        <li class="cc-compaign"><a href="#about" class="cc_link">CC compain</a></li>
+    </div>
+      
+    </nav>
+  </header>
+    <main>
+        <div class="introduction">
+        <p class="topic">Welcome to sport Festival</p>
+        <h2 class="topic2">Sport Festival 2022</h2>
+        <p class="main-paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia ipsum aut distinctio aspernatur exercitationem? Cum quis facere natus accusantium perspiciatis, fugit at quasi in excepturi officia eaque distinctio nemo beatae? Lorem ipsum, dolor sit amet consectetur adipisicing elit. Vero at dolorum neque iusto sit deserunt excepturi repellat porro dolor omnis quas, tempora magni nam. Unde eligendi consequuntur placeat totam ipsa?</p>
+        <div class="footer">
+        <p class="date">2023.07.19(THU) <p>
+           <p class="address"> @ Macroyan, Kabul, Afghanistan</p>
+        </div>
```

Comment:
- [ ] Please try to improve the styles in this hero section to match the design given 👍 Look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/205446656-95fa2a9a-407c-43b1-80f1-d0081d17563c.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/205446683-efba8f85-feef-4124-b7c5-5e07aaba4a2b.png)

As you can see there is a lot to be improved. Please improve it correctly